docs/agents/registry.rst

```diff
@@ -6,36 +6,45 @@ Registry Agent
 =======================
 
 The registry agent is used to keep track of currently running active agents.
-Agents are automatically registered OnJoin, and the registry monitors
-each agent's heartbeat to remove the agent if it disconnects.
+It listens to the heartbeat feeds of all agents on the crossbar server, 
+and keeps track of the last heartbeat time of each agent and whether 
+or not each agent has agent has "expired" (gone 5 seconds without a heartbeat).
 
-One can keep track of active agents by monitoring the *agent_activity*
-feed, which will publish whenever an agent is registered, unregistered,
-or updated. Calling the *dump_agents* task will print the current status
-of all active agents to the feed.
+This check happens in the registry's single "run" process. The session.data object
+of this process is set to a dict of agents on the system, including their last 
+heartbeat time, whether they have expired, and the time at which they expired.
+This data can the be viewed by checking the session variable of the run process.
 
-Docker Configuration
+For instance, the following code will print agent's that have been on the system
+since the registry started running::
+
+    from ocs.matched_client import MatchedClient
+
+    registry_client = MatchedClient('registry')
+    status, msg, session = registry_client.run.status()
+
+    print(session['data'])
```

Comment:
Somewhere you need to show an example of what this session.data looks like.  It could be here, or in the docstring for the operation (to keep it close to the implementation) but referenced from here.